fluentlenium-core/src/main/java/org/fluentlenium/core/Fluent.java

```diff
@@ -360,6 +360,32 @@ public Fluent goTo(String url) {
         return this;
     }
 
+    /**
+     * Open the url page in a new tab
+     *
+     * @param url
+     */
+    public Fluent goToInNewTab(String url) {
+        if (url == null) {
+            throw new IllegalArgumentException("Url is mandatory");
+        }
+
+        if (baseUrl != null) {
+            URI uri = URI.create(url);
+            if (!uri.isAbsolute()) {
+                url = baseUrl + url;
+            }
+        }
+
+        Set<String> initialTabs = getDriver().getWindowHandles();
+        executeScript("window.open('" + url + "', '_blank');");
```

Comment:
I'm not sure is it the best option to open new tab. It might be but I need to ask you, have you tested your code on IE, Firefox, Chrome and on Safari as well?

Here are another options to deal w/ that: http://stackoverflow.com/questions/17547473/how-to-open-a-new-tab-using-selenium-webdriver
I'm just making sure, please confirm or test your code if you can.